modules/openapi-generator/src/main/java/org/openapitools/codegen/languages/SpringCodegen.java

```diff
@@ -1196,4 +1198,38 @@ public RequestMappingMode getRequestMappingMode() {
     public void setRequestMappingMode(RequestMappingMode requestMappingMode) {
         this.requestMappingMode = requestMappingMode;
     }
+
+    @Override
+    public CodegenParameter fromParameter( final Parameter parameter, final Set<String> imports ) {
+        CodegenParameter codegenParameter = super.fromParameter( parameter, imports );
+        if( !codegenParameter.isContainer || codegenParameter.isMap){
+            return codegenParameter;
+        }
```

Comment:
No Map is more complicated for Bean validation, because you need maybe both generic types.  Found no easy solution for that
e.g.
Map<@Valid Bean, String>
Map<String, @Valid Bean>
Map<@Valid Bean, @Valid Bean>

Reply:
@borsch do you also have a good idea for this issue or should I add a second ticket for that?
I think bean validation with Maps is not so popular. So I could skip this for this PR

Reply:
@welshm @MelleD map is a pretty complex scenario. Probably we can go with `Map<@Valid T, @Valid K>` and it should be fine

Reply:
@borsch you mean to add it every time so for example above:

```
Map<@Valid Bean, @Valid String>
Map<@Valid String, @Valid Bean>
Map<@Valid Bean, @Valid Bean>
```

I think it make no big difference it's just ugly to read :).

Reply:
yep, that's true :)
therefore might be make sense to leave it as it is as for now

Reply:
Ok i make a comment what the issue is with Map and make it clearer.
Somebody a hint how can i fix my codestyle issue? Is there a code style xml available?